File: ClinicInfo_API/ClinicInfo.API/ClinicInfo.API/Entities/Patient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ClinicInfo.API.Entities
{
    public class Patient
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int PatientId { get; set; }

        [Required]
        [MaxLength(50)]
        public string Name { get; set; }

        [Required]
        [MaxLength(50)]
        public string Surname { get; set; }

        public string Image { get; set; }

        public double Balance { get; set; }

        public ICollection<Transaction> Transactions { get; set; } = new List<Transaction>();

        public ICollection<PatientService> PatientServices { get; set; } = new List<PatientService>();

        public ICollection<Notification> Notifications { get; set; } = new List<Notification>();

    }
}

File: primengSetup/prime-try/src/app/patient/patient.ts
export interface IPatient {

    patientId : number;
    name : string;
    surname : string;
    image : string;
    balance : number;
}

export class Patient implements IPatient{

    constructor(
        public patientId: number,
        public name: string,
        public surname: string,
        public image: string,
        public balance: number){

    }
}